include/odp/api/spec/classification.h

```diff
@@ -204,6 +209,11 @@ typedef struct odp_cls_capability_t {
 
 	/** Supported threshold type for BP */
 	odp_threshold_types_t threshold_bp;
+
+	/** Maximum mark value.
+	 * odp_pmr_create_opt_t::mark value must be less than this value.
```

Comment:
So now max_mark would not be maximum mark value (but maximum mark value plus one). That is unclear naming and there is no good reason for the "less than" requirement.

I would rephrase: "maximum value of odp_pmr_create_opt_t::mark".

Reply:
Make sense. I will fix it in next version.